src/clientManager.js

```diff
@@ -3,13 +3,31 @@ var fs = require("fs");
 var Client = require("./client");
 var mkdirp = require("mkdirp");
 var Helper = require("./helper");
+var path = require("path");
+
+var USERS_PATH = path.join(Helper.HOME, "users");
```

Comment:
You can directly assign it to `ClientManager.USERS_PATH`, and in this file use `this.USERS_PATH`